_includes/projects-gallery.html

```diff
@@ -1,8 +1,7 @@
 <div class="gallery-container">
-  {% assign filtered_posts = site.posts | where: 'layout', 'project' %} {% for
-  post in filtered_posts %}
-  {% include card.html thumbnail=post.thumbnail description=post.description url=post.url alt=post.alt areas=post.areas title=post.title imgtext=post.imgtext %}
-  {% endfor %}
+  {% assign filtered_posts = site.posts | where: 'layout', 'project' %}
+  {% assign class = "non-alternating" | append: include.class %}
+  {% include card-group.html cards = filtered_posts class = "non-alternating" %}
```

Comment:
same here, I think since class = "non-alternating" on line 4, we might not need line 3

Reply:
Good catch!